docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -152,6 +153,12 @@ Next, you can run a simple example program:
 $ bin/alluxio runTests
 ```
 
+For this test to succeed, you need to make sure that the login user of Alluxio cli, i.e. `bin/alluxio` has
```

Comment:
I think you can remove the part which mentions `bin/alluxio`.

Reply:
Done.